src/components/actions/evaluatorFunctions.ts

```diff
@@ -10,15 +10,26 @@ const generateExpiry = (duration: Duration) => DateTime.now().plus(duration).toJ
 const getYear = (type?: 'short'): string =>
   type === 'short' ? String(new Date().getFullYear()).slice(2) : String(new Date().getFullYear())
 
+type FormatDate =
+  | string
+  | {
+      format: string
+      locale: string
+    }
+
 // Returns ISO date string or JS Date as formatted Date (Luxon). Returns current
 // date if date not supplied
-const getFormattedDate = (formatString: string, date?: string | Date) =>
-  (date
+const getFormattedDate = (formatString: FormatDate, date?: string | Date) => {
+  const tempDate = date
     ? typeof date === 'string'
       ? DateTime.fromISO(date)
       : DateTime.fromJSDate(date)
     : DateTime.now()
-  ).toFormat(formatString)
+
+  if (typeof formatString === 'string') return tempDate.toFormat(formatString)
+  const { format, locale } = formatString
+  return tempDate.setLocale(locale).toFormat(format)
```

Comment:
```suggestion
  return date.toFormat(format, { locale })
```
Also nit, but I like this syntax more. Makes it more obvious that locale can be optional, plus concepturally it's really only one operation with two parameters.